using-valtimo/command-handling/command-handling.md

```diff
@@ -0,0 +1,69 @@
+# Command handling
+
+A command is used to change the state of an application. Commands allow you to express clear intentions which simplify understanding and debugging of the application.
```

Comment:
Can you move the documentation to Extending Valtimo rather than Using Valtimo? As this requires custom code.